client/my-sites/plans/jetpack-plans/build-card-features-from-item.ts

```diff
@@ -10,9 +10,8 @@ import type { Plan, Feature } from '@automattic/calypso-products';
 
 /**
  * Builds the feature item of a product card, from a feature key.
- *
- * @param {Feature[]|Feature} featureKey Key of the feature
- * @returns {SelectorProductFeaturesItem} Feature item
+ * @param {Feature} featureKey Key of the feature
```

Comment:
Same as above - for typed arguments we should be able to remove the types from JSDoc.